content/articles/srv-record.md

```diff
@@ -25,19 +25,32 @@ Here's an example of two SRV records.
     _sip._tcp.example.com.   3600 IN    SRV 10       60     5060 bigbox.example.com.
     _sip._tcp.example.com.   3600 IN    SRV 10       20     5060 smallbox1.example.com.
 
-From the name, `_sip` is the symbolic name for the service and `_tcp` is the transport protocol. The symbolic name and transport always start with an underscore.
+`_sip` is the symbolic name for the service and `_tcp` is the transport protocol. The symbolic name and transport protocol always start with an underscore.
 
-The content of the SRV record defines a priority of 10 for both records. The first record has a weight of 60 and the second a weight of 20. The priority and weight values can be used to encourage use of certain servers over others.
+The content of the SRV record defines a priority of 10 for both records. The first record has a weight of 60, and the second has a weight of 20. The priority and weight values can be used to encourage use of certain servers over others.
 
 The final two values in the record define the port and hostname to connect to for accessing the service.
 
+## Adding an SRV record
 
-## Add an SRV record
+To add an SRV record in DNSimple:
 
-To add an SRV record in DNSimple, you might fill in the new SRV record form like this:
+1. On your Domain List, click the relevant domain. 
+1. Click the **DNS** badge at the top right to access the [Record Editor](/articles/record-editor/).
+
+<!--- needs screenshot -->
```

Comment:
https://drive.google.com/file/d/1-lmjzhqCcbf9-F-9Ur9z8PzAVe73wn95/view?usp=sharing

Reply:
it looks like this image is also borking the numbering 🫠